Dockerfile

```diff
@@ -77,4 +77,8 @@ ONBUILD COPY src/Pyz $WORKDIR/src/Pyz
 ONBUILD COPY config $WORKDIR/config
 ONBUILD COPY public $WORKDIR/public
 ONBUILD RUN /entrypoint.sh build-shop
+ONBUILD COPY codeception.yml $WORKDIR/
+
+ONBUILD COPY setup_test $WORKDIR/
```

Comment:
Yes there are. Using `*test*`, all the contents of `tests/` is copied into the root of the *WORKDIR*.